docker-compose.yml

```diff
@@ -94,9 +94,9 @@ services:
       - ~/.ivy2:/root/.ivy2:cached
       - ~/.m2:/root/.m2:cached
       - ~/.sbt:/root/.sbt:cached
-      - ~/.cache/coursier:/root/.cache/coursier:cached
+      - ~/.cache/coursier:/home/sbtuser/.cache/coursier:cached
```

Comment:
Why was this done? On my system this would break the coursier cache causing SBT to redownload dependencies all the time

Reply:
```
C:\Users\vorop\IdeaProjects\quill>docker exec -it quill_sbt_run_26a3921e85d6  /bin/bash
root@794685cb480a:/app# ls -la /home/sbtuser/
total 20
drwxr-xr-x 1 sbtuser sbtuser 4096 Sep 25  2019 .
drwxr-xr-x 1 root    root    4096 Sep 25  2019 ..
drwxr-xr-x 1 sbtuser sbtuser 4096 Jul 11 15:05 .cache
drwxrwxrwx 1 root    root     512 Jun 14 20:32 .ivy2
drwxrwxrwx 1 root    root     512 Jun 14 20:33 .sbt
root@794685cb480a:/app# du -sk /home/sbtuser/.cache
780824  /home/sbtuser/.cache
root@794685cb480a:/app# du -sk /home/sbtuser/.ivy2/
194642  /home/sbtuser/.ivy2/
root@794685cb480a:/app# du -sk /home/sbtuser/.sbt/
301750  /home/sbtuser/.sbt/

root@794685cb480a:/app# ls -l /root/
total 0

```
On my system its the opposite.

Reply:
Ah the user for the sbt-docker image must have been updated in a recent PR, in that case can you please update all of the entires from `/root`/ to `/home/sbtuser` and not just line 97?

Reply:
will do.
It the the case for all except for `.m2`

Reply:
I'm concerned about this. How come all of the build paths are changing? Did we bump up the image of something?

Reply:
@deusaquilus I bumped the sbt-docker image to co-incide with the latest sbt-image and I suspect that the latest versions of the sbt-docker image have a seperate non root user (which here is called `sbt-user`) instead of having everything inside of root.

Reply:
So I just ran `docker-compose run --rm sbt whoami` on my system and I got back `root` so I have no idea where `sbt-user` is coming from.

@voropaevp Can you rebase this PR off of master (to make sure the PR is in sync) and run `docker-compose run --rm sbt whoami` and tell me the output? Alternately we can just update all of this SBT stuff in a separate PR so we don't mix too many concerns.

Reply:
```
root@314098d77a1d:/app# ls -la /root
total 20
drwx------ 1 root root 4096 Jul 11 21:01 .
drwxr-xr-x 1 root root 4096 Jul 11 21:01 ..
-rw-r--r-- 1 root root  570 Jan 31  2010 .bashrc
lrwxrwxrwx 1 root root   20 Sep 25  2019 .cache -> /home/sbtuser/.cache
lrwxrwxrwx 1 root root   19 Sep 25  2019 .ivy2 -> /home/sbtuser/.ivy2
drwxrwxrwx 1 root root  512 Jun 21 00:34 .m2
-rw-r--r-- 1 root root  148 Aug 17  2015 .profile
lrwxrwxrwx 1 root root   18 Sep 25  2019 .sbt -> /home/sbtuser/.sbt
-rw-r--r-- 1 root root  165 Sep 14  2019 .wget-hsts
root@314098d77a1d:/app#  whoami
root
```

Reply:
Reverted all the changes with exception for ReservedCodeCacheSize.